.github/workflows/ci.yaml

```diff
@@ -57,20 +57,3 @@ jobs:
         env_vars: OS,PYTHON
         name: codecov-umbrella
         fail_ci_if_error: false
-
-  distribution:
-    runs-on: ubuntu-latest
-    needs: [pytest]
-
-    steps:
-    - uses: actions/checkout@v3
-    - name: Build distributions
-      run: |
-        $CONDA/bin/python -m pip install build
-        $CONDA/bin/python -m build
-    - name: Publish a Python distribution to PyPI
-      if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
-      uses: pypa/gh-action-pypi-publish@release/v1
-      with:
-        user: __token__
-        password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
@malmans2 let me know if this is OK. I factor it out so we can include sdist and wheels only tests in a new GHA.